crates/anstyle-parse/src/state/definitions.rs

```diff
@@ -2,6 +2,7 @@ use core::mem;
 
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
 #[repr(u8)]
+#[derive(Default)]
```

Comment:
This commit and PR do too much.  The clippy changes are unrelated to the version req changes.  Please revert this

Reply:
Okay, I am sorry.  I run all usual Cargo checks to ensure that I did not break something and Clippy suggested these changes to me.  I thought it would be useful to apply these suggestions.